dev-requirements.txt

```diff
@@ -3,3 +3,4 @@ flake8
 testpath
 setuptools>=30
 tomli >=1.1.0 ; python_version<'3.11'
+importlib-metadata; python_version<'3.8'
```

Comment:
Not sure if this is needed, given that we are skipping the test when `importlib.metadata` is not available. Probably not (I probably should remove this)...